site/tutorials/keptn-litmus-08.md

```diff
@@ -86,23 +86,88 @@ Duration: 3:00
     kubectl apply -f ./litmus/pod-delete-rbac.yaml 
     ```
 
-## Setup Prometheus
-Duration: 2:00
+## Setup Prometheus Keptn integration
 
-Before we are going to create the project with Keptn, we'll install the Prometheus integration to be ready to fetch the data that is later on needed for the SLO-based quality gate evaluation. 
+Duration: 3:00
 
-1. The Prometheus service integration is responsible to set up Prometheus with Keptn, let us install the integration. Please note, this does **not** automatically install Prometheus - this will be done later in the tutorial.
+After creating a project and service, you can set up Prometheus monitoring and configure scrape jobs using the Keptn CLI.
 
-    ```
-    kubectl apply -f https://raw.githubusercontent.com/keptn-contrib/prometheus-service/release-0.4.0/deploy/service.yaml
-    ```
+Keptn doesn't install or manage Prometheus and its components. Users need to install Prometheus and Prometheus Alert manager as a prerequisite.
 
-1. Next, we are going to install the Prometheus SLI integration to be able to fetch the data from Prometheus
+* To install the Prometheus and Alert Manager, execute:
 
-    ```
-    kubectl apply -f https://raw.githubusercontent.com/keptn-contrib/prometheus-sli-service/release-0.3.0/deploy/service.yaml
-    ```
+```bash
+kubectl create ns monitoring
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm install prometheus prometheus-community/prometheus --namespace monitoring
+```
+
+**Execute the following steps to install prometheus-service**
+
+* To install the prometheus-service, execute:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/keptn-contrib/prometheus-service/release-0.5.0/deploy/service.yaml
+```
+
+* Edit the prometheus-service deployment and patch it with these environment variables:
+
+```bash
+kubectl edit deploy prometheus-service -n keptn
```

Comment:
we can make this easier for the user with the following commands:
```
kubectl set env deployment/prometheus-service -n keptn --containers="prometheus-service" PROMETHEUS_NS=monitoring
kubectl set env deployment/prometheus-service -n keptn --containers="prometheus-service" ALERT_MANAGER_NS=monitoring
```

Reply:
with this we can ensure that users do not make a typo as they can copy and paste both commands.

Reply:
Please update the text, with something like:

Next, we are going to set the namespace that Prometheus has been deployed (i.e., the `monitoring` namespace) in the environment variables section of Keptn's `prometheus-service`.